sklearn/utils/_metadata_requests.py

```diff
@@ -59,12 +59,12 @@
 
 To give the above representation some structure, we use the following objects:
 
-- ``(caller, callee)`` is a namedtuple called ``MethodPair``
+- ``(caller=..., callee=...)`` creates an instance of a ``MethodPair`` class
```

Comment:
it's a `namedtuple`, not a usual class.

Reply:
Can we say it's a `tuple` that is used to create a such a class?
I admit it's not a class just yet (not in the moment when it's added to a MetadataRouter).
But I don't think it's a `namedtuple() `function either.

Reply:
It's not a `namedtuple` function, `namedtuple` returns something which is `MethodPair`. So `MethodPair` is a namedtuple. It's certainly not a normal `tuple`.

Reply:
I'm not sure if I follow entirely, though I will revert this part to how it was before.